demo/vl-pattern.html

```diff
@@ -41,6 +42,11 @@
                 <label is="vl-form-label" for="pattern-5">Rijksregisternummer</label>
                 <input is="vl-input-field" data-vl-block type="text" id="pattern-rrn" name="pattern-5" value="" placeholder="11.22.33-444.55" data-vl-pattern="rrn" data-vl-js-dress="true" />
             </vl-demo>
+
+            <vl-demo data-vl-title="UUID validation">
+                <label is="vl-form-label" for="pattern-6">UUID</label>
```

Comment:
een reden waarom pattern-6 en niet pattern-uuid?

Reply:
in de demo erboven staat er ook iets gelijkaardigs 'pattern-5', zegt niet zoveel natuurlijk

Reply:
Heb het aangepast